third_party/v8/src/wasm/wasm-code-manager.cc

```diff
@@ -178,14 +178,14 @@ std::unique_ptr<const byte[]> WasmCode::ConcatenateBytes(
   size_t total_size = 0;
   for (auto& vec : vectors) total_size += vec.size();
   // Use default-initialization (== no initialization).
-  byte* result = new byte[total_size];
```

Comment:
If this is the original code, it had problems building on some of the 1P platforms.